src/OrbitQt/MoveFilesDialog.cpp

```diff
@@ -33,4 +33,20 @@ void MoveFilesDialog::EnableCloseButton() {
   ui_->closeButton->setEnabled(true);
 }
 
+void MoveFilesDialog::closeEvent(QCloseEvent* event) {
+  if (ui_->closeButton->isEnabled()) {
+    event->accept();
+  } else {
+    event->ignore();
+  }
+}
+
+void MoveFilesDialog::keyPressEvent(QKeyEvent* event) {
+  if (event->key() != Qt::Key_Escape || ui_->closeButton->isEnabled()) {
```

Comment:
ok, I see... but in this case does closeEvent do anything?

Reply:
`closeEvent` is called when the window is closed with the X or the equivalent in some menu, that's what it prevents. But not on ESC.